doc/md/declarative/apply.mdx

```diff
@@ -0,0 +1,46 @@
+---
+id: apply
+slug: /declarative/apply
+title: Declarative schema migrations
+---
+
+import Help from '../generated/_atlas_schema_apply.mdx';
+
+With Atlas, users do not need to plan database schema changes themselves. Instead
+of figuring our the correct SQL statements to get their database to the desired schema,
+declarative schema migrations only require the user to provide the desired schema and
+a connection string to the target database, and can get a migration plan that is tailor-made
+to the specific database engine they are working against.
```

Comment:
This whole sentence is a little long. Break it up into two/three, something like: "Rather than figuring out the correct SQL statements to get their database to the desired state, declarative migrations offer a straightforward solution. With declarative migrations, the user is only required to provide the desired schema and a connection string to the target database. Atlas generates a tailor-made migration plan that is specific to the database engine the user is working against."